deploy/00_resolve_keep_token_staking.ts

```diff
@@ -11,10 +11,15 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   if (KeepTokenStaking && helpers.address.isValid(KeepTokenStaking.address)) {
     log(`using existing KeepTokenStaking at ${KeepTokenStaking.address}`)
   } else if (
-    hre.network.name !== "hardhat" ||
+    hre.network.name === "hardhat" &&
```

Comment:
Same comments as in `deploy/00_resolve_keep_registry.ts`.

Reply:
pls see above